docs/docs/metrics.md

```diff
@@ -107,10 +111,9 @@ Dashes in metric names are replaced with underscores.
   compaction durations.
 * `marathon.persistence.gc.scan.duration.timer.seconds` — a histogram of
   Marathon GC scan phase durations, and a meter for scan durations.
-* `marathon.instances.running.gauge` — the number of running instances
-  at the moment.
-* `marathon.instances.staged.gauge` — the number of instances staged at
-  the moment.
+* `marathon.pods.active.gauge` — the number of active pods.
```

Comment:
Where did the `marathon.instances.running.gauge` go? Why did `marathon.pods.active.gauge` replace it?

Reply:
I just sorted it alphabetically. It was moved up a bit in this list.